products/colp/metadata.yml

```diff
@@ -0,0 +1,282 @@
+name: COLP
+display_name: City Owned and Leased Property (COLP)
+summary: |
+  City Owned and Leased Properties (COLP) is published by the Department of City Planning (DCP) as required by chapter 204 of the City Charter. It includes the location and current use of all City-owned and leased properties.
+description: |
+  City Owned and Leased Properties (COLP) is published by the Department of City Planning (DCP) as required by chapter 204 of the City Charter. It includes the location and current use of all City-owned and leased properties. COLP is produced from an extract of the Integrated Property Information System (IPIS), a real estate database maintained by the Department of Citywide Administrative Services (DCAS). Except where indicated, the data provided is from IPIS.
+
+  Questions or feedback on errors within the City Owned and Leased Properties can be directed to DCAS Land Use Planning at (212) 386-0622 or RESPlanning311@dcas.nyc.gov.
+
+tags: [COLP, IPIS, city owned, leased, property, Department of Citywide Administrative Services, DCAS, Department of City Planning, DCP, New York City, NYC]
+
+columns:
+  - name: uid
+    display_name: Unique ID
+    data_source: Department of City Planning
+    description: >-
+      Unique identifier created from a hash of all fields for the record on the
+      IPIS source file. As long as no fields change on the record, this
+      identifier is static between versions of COLP.
+    data_type: text
+    example: cbe20732be28f6ab445289d7a67bb241
+
+  - name: BOROUGH
+    display_name: Borough
+    description: >-
+      NYC borough – 1 (Manhattan), 2 (Bronx), 3 (Brooklyn), 4 (Queens), 5
+      (Staten Island)
+    data_type: text
+    is_nullable: False
+    values:
+      - ["1", Manhattan]
+      - ["2", Bronx]
+      - ["3", Brooklyn]
+      - ["4", Queens]
+      - ["5", Staten Island]
+
+  - name: BLOCK
+    display_name: Tax block
+    description: >-
+      The tax block in which the tax lot is located. Each tax block is unique
+      within a borough.
+    data_type: integer
+    example: 1637
+
+  - name: LOT
+    display_name: Tax lot
+    description: The number of the tax lot. Each tax lot is unique within a tax block.
+    data_type: integer
+    example: 141
+
+  - name: BBL
+    display_name: BBL
+    description: >-
+      10-digit identifier for a tax lot, consisting of the borough code followed
+      by the tax block followed by the tax lot. The borough code is one numeric
+      digit. The tax block is one to five numeric digits, preceded with leading
+      zeros when the block is less than five digits. The tax lot is one to four
+      digits and is preceded with leading zeros when the lot is less than four
+      digits. For condominiums, this is usually the unit BBL.
+    data_type: bbl
+    data_readme_type: double
```

Comment:
thoughts about this name? This is for cases where we have a concrete type that we can validate (e.g. a BBL), but what appears in the README currently is a less specific type, like `double`.

Reply:
slight preference for `readme_data_type` but no big deal

if we aren't validating READMEs (not suggesting we do), do we need this field? maybe near future for generating READMEs?

Reply:
@damonmcc cool, yeah I agree. Will change in the followup PR